connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/application/ReplE2ESuite.scala

```diff
@@ -134,20 +134,6 @@ class ReplE2ESuite extends RemoteSparkSession with BeforeAndAfterEach {
     assertContains("Array[Int] = Array(19, 24, 29, 34, 39)", output)
   }
 
-  // SPARK-43198: Switching REPL to CodeClass generation mode causes UDFs defined through lambda
```

Comment:
This will never work. See [SPARK-29497](https://issues.apache.org/jira/browse/SPARK-29497)